### DISCUSSION TITLE
Plans for viewing and editing Gitlab issues

### DISCUSSION THREAD

**Discussion by tnthi115** [harrisoncramer/gitlab.nvim#310] [Ideas]
Hi @harrisoncramer, are there are any plans for viewing and editing Gitlab issues, similar to what [octo.nvim](https://github.com/pwntester/octo.nvim?tab=readme-ov-file) does for Github?

**Comment by harrisoncramer** [comment]
Hey there,

Having issue support would be awesome, but it's not on my roadmap because 1) it's a lot of work, and 2) it's not something my company uses. That said, if someone in the community is interested in working on this I'd be happy to guide/review them. It'd probably involve a lot of net-new code since the plugin is almost entirely centered around merge requests right now, not issues.
